SQL/Archive/17.1/2016-11-21-CreateUserSitesRelTable.sql

```diff
@@ -0,0 +1,21 @@
+WARNINGS;
```

Comment:
Thanks for explanation. I got a console error when running this on GUI client, so that's why I was confused